lib/uploadcare/api/file_storage_api.rb

```diff
@@ -0,0 +1,29 @@
+module Uploadcare
+  module FileStorageApi
+    BATCH_SIZE = 100
+
+    def store_files(objects)
+      in_batches(objects) { |batch| put "/files/storage/", to_uuids(batch) }
+    end
+
+    def delete_files(objects)
+      in_batches(objects) { |batch| delete "/files/storage/", to_uuids(batch) }
+    end
+
+    private
+
+    def in_batches(enum)
+      enum.each_slice(BATCH_SIZE) { |batch| yield batch }
+    end
+
+    def to_uuids(objects)
+      objects.map do |object|
+        case object
+        when Uploadcare::Api::File then object.uuid
+        when /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/ then object
+        else raise(ArgumentError, "Unable to convert object to uuid: #{object}")
```

Comment:
@dmitry-mukhin then I'll make this new methods to return problems hash if there were any problems of true otherwise:

```ruby
api.store_files(uuids_array) # => true
api.store_files(['garbage1', 'garbage2']) # => {"garbage1" => "Invalid", "garbage2" => "Invalid"}
```

If > 100 objects are being stored/deleted at once, then there will be multiple store/delete requests. In this case problems hash will be an aggregation of problems hashes from all responses with problems.